lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -89,7 +89,7 @@ def ss_alt_fees_table_country?(country_slug, calculator)
 
     def ss_marriage_not_possible?(country_slug, calculator)
       (SS_ALT_FEES_TABLE_OR_OUTCOME_GROUP_A.include?(country_slug) && calculator.partner_not_british?) ||
-        ((SS_ALT_FEES_TABLE_OR_OUTCOME_GROUP_B.include?(country_slug) || %w(cambodia vietnam).include?(country_slug)) && calculator.partner_is_national_of_ceremony_country?) ||
+        ((SS_ALT_FEES_TABLE_OR_OUTCOME_GROUP_B.include?(country_slug) || %w(cambodia).include?(country_slug)) && calculator.partner_is_national_of_ceremony_country?) ||
```

Comment:
well this affects only same sex marriages where the partner is vietnamese.. and yes it does produce artefacts.. I will demotrate that in its own commit..